tests/README.md

```diff
@@ -1,8 +1,10 @@
-To run tests all tests you will need to run the `kennethreitz/httpbin` and `ghcr.io/chatterino/twitch-pubsub-server-test:latest` docker images
+To run tests all tests you will need to run the `kennethreitz/httpbin` and `ghcr.io/chatterino/twitch-pubsub-server-test:latest` docker images.
 
 For example:
 
 ```bash
 docker run --network=host --detach ghcr.io/chatterino/twitch-pubsub-server-test:latest
 docker run -p 9051:80 --detach kennethreitz/httpbin
 ```
+
+Alternatively, you can use [httpbox](github.com/kevinastone/httpbox) (`httpbox --port 9051`).
```

Comment:
Phrase this as: "If you're unable to use docker..." & include a link to https://github.com/Chatterino/twitch-pubsub-server-test/releases/latest too